Documentation/project-docs/glossary.md

```diff
@@ -139,3 +139,7 @@ The .NET platform currently doesn't contain a cross-platform XAML-based UI stack
 * [WPFLight](https://github.com/ronnycsharp/WPFLight): An OSS project to create WPF on top of XNA/MonoGame.
 * [Perspex](https://github.com/Perspex/Perspex): A cross-platform UI framework based on WPF.
 * [Granular](https://github.com/yuvaltz/Granular): A OSS project to allow WPF applications to run in the browser.
+
+## Engineering system
+
+* **Helix**. Microsoft internal massive parallel job execution system used to run [xunit](http://xunit.github.io/) tests. Tests result are shown through *Mission Control* reporting site https://mc.dot.net/
```

Comment:
Nit: Helix can run arbitrary workitems, it is not limited to xunit tests. It is a general purpose job processing and result aggregation system running in cloud. It just happens that the workitems that corefx sends to Helix are xunit tests.

Reply:
We actually just changed package tests to run in helix as well which are not xunit tests, but package validation for all of its supported frameworks and RIDs :smile: